docs/book/unit-testing.md

```diff
@@ -27,7 +27,7 @@ integration for laminas-mvc, including application scaffolding and custom
 assertions. You will need to install it:
 
 ```bash
-$ composer require --dev laminas/laminas-test phpunit/phpunit
+$ composer require --dev laminas/laminas-test phpunit/phpunit phpspec/prophecy-phpunit
```

Comment:
I suggest removing `phpspec/prophecy-phpunit` from the affected code, instead of introducing it in the tutorial.